### DISCUSSION TITLE
What file formats are supported?

### DISCUSSION THREAD

**Discussion by LucaMkIII** [FossifyOrg/General-Discussion#566] [Q&A] [answered]
What file formats are supported? Or, is there a place I can reference that has a list of supported formats?

This came up because I was trying to play some .amr files and getting an error. I thought they might not be supported, so I looked for a list, but couldn't find one. (If .amr files *are* supported, there may be a bug with my device or the files.)

**Comment by Aga-C** [comment, marked as answer]
Here is a list of supported file types: mp3, wav, wma, ogg, m4a, opus, flac, aac and m4b.

**Reply by inson1** [reply]
that should be in readme imo

**Reply by LucaMkIII** [reply]
@Aga-C Thank you!

@inson1 I agree, either in the readme or in the FAQs (if this question gets asked frequently).

**Reply by inson1** [reply]
I guess this should be in feature section which isnt implemented yet
